<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="ListTask">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>760</width>
      <height>350</height>
     </rect>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="gridStyle">
     <enum>Qt::PenStyle::SolidLine</enum>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="MarkDoneButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>410</y>
      <width>360</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Mark Done ‚úÖ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DeleteButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>360</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Task üóëÔ∏è</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AddTaskButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>625</y>
      <width>760</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>+ ADD TASK</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>520</y>
      <width>360</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>760</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Enter Task Name</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>360</width>
      <height>32</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>One-time Task</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
